src/components/Modal/Modal.scss

```diff
@@ -0,0 +1,104 @@
+
+.af-modal-wrapper {
+  font-family: Aktiv Grotesk, -apple-system, BlinkMacSystemFont, San Francisco, Segoe UI, Roboto, Helvetica Neue, sans-serif;
+  position: absolute;
+  z-index: 1000;
+  width: 100%;
+  height: 100%;
+  left: 0;
+  top: 0;
+
+  &.--hidden {
+    display: none;
+  }
+}
+
+.af-modal-overlay {
+  background: rgba(0, 0, 0, 0.6);
+  width: 100%;
+  height: 100%;
+  position: fixed;
+}
+
+.af-modal {
+  position: fixed;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  top: 50%;
+  right: 0;
+  bottom: 0;
+  display: flex;
+  z-index: 2;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  max-width: 43.75rem;
+  max-height: 43.75rem;
+
+  &__content {
+    width: 100%;
+    height: 100%;
+    background-color: #fff;
+  }
+
+  &__header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 0.6rem 1.2rem;
+  }
+
+  &__header-left-corner {
+    width: 20%;
+  }
+
+  &__close-button {
+    color: #2d2f34;
+    border: none;
+    background: transparent;
+    font-size: 1.45rem;
+    cursor: pointer;
+  }
+
+  &__header-center {
+    width: 60%;
+  }
+
+  &__title {
+    margin: 0 auto;
+    text-align: center;
+    color: #828282;
+    font-size: 1.1rem;
+    font-weight: 500;
+    text-transform: uppercase;
+  }
+
+  &__header-right-corner {
+    width: 20%;
+    text-align: end;
+  }
+
+  &__corp {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+  }
+
+  &__body {
+    border-top: 2px solid #e2e2e2;
+    text-align: center;
+    overflow: scroll;
```

Comment:
`overflow: scroll;` will add a permanent scroll bar even if the content is not overflowing. Check the overflow documentation to learn what value can you give it, so that the scrollbar only shows if the content overflows